src/main/java/com/google/opensesame/util/RateLimitExceededException.java

```diff
@@ -0,0 +1,23 @@
+package com.google.opensesame.util;
+
+import java.io.IOException;
+
+/**
+ * The exception to be thrown when the GitHub API has run out of calls. This will help us avoid
+ * lengthy stalls when we lose access to GitHub Data i.e. Our API calls have exceeded our current
+ * GitHub hourly call rate limit.
```

Comment:
nit: I don't think this information is 100% relevant here. Consider removing this comment, as this information is already present in a more relevant spot on the `GitHubRateLimitHandler`.
```suggestion
 * The exception to be thrown when the GitHub API has run out of calls.
```